app/frontend/css/cdo-task-list.scss

```diff
@@ -0,0 +1,56 @@
+.defra {
```

Comment:
Having ready through fully now, I realise this is for the Dog/Dog Record (button)/Owner record (button) section. It will therefore be needed on the 'View dog details' page so can we change the name to be more generic, such as action-page-header or something?